core/css/public.scss

```
@@ -3,19 +3,7 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 #body-public {
```

I would keep the default value as an initial value before the script is initiated (can take some time depending on your internet connection and loading order) and to help understand that this value exists ;)

```suggestion
#body-public {
    // This value is updated by JavaScript to match the footers actual size
    --footer-height: calc(2lh + 2 * var(--default-grid-baseline));
```